src/java/dynamicstructures/SinglyLinkedList.java

```diff
@@ -0,0 +1,76 @@
+package dynamicstructures;
+
+public class SinglyLinkedList {
+    private Node head;
+
+    private class Node {
+        private int data;
+        private Node next;
+
+        private Node(int data) {
+            this.data = data;
+
+        }
+    }
+
+    public void addElement(int data) {
+        Node newNode = new Node(data);
+
+        if (head == null) {
+            head = newNode;
+        } else {
+            Node current = head;
+            while (current.next != null) {
+                current = current.next;
+            }
+            current.next = newNode;
+        }
+    }
+
+
+    public void displayList() {
+        Node current = head;
+
+        while (current != null) {
+            System.out.println(current.data);
+            current = current.next;
+        }
+    }
+
+    public void removeDuplicates() {
```

Comment:
You can rename this method to be called removeDuplicatesSortedList, to avoid confusion